src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/makerecalldecisions/recommendation/AlternativesToRecallTried.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.recommendation
+
+data class AlternativesToRecallTried(
+  val selected: List<SelectedAlternative>?,
+  val allOptions: List<RecallAlternative>?
```

Comment:
You can use `List<TextValueOption>?` here and remove your `RecallAlternative` class

Reply:
Done